src/cargo/core/compiler/mod.rs

```diff
@@ -140,10 +140,11 @@ pub trait Executor: Send + Sync + 'static {
 pub struct DefaultExecutor;
 
 impl Executor for DefaultExecutor {
+    #[instrument(name = "rustc", skip_all, fields(package = id.name().as_str(), process = cmd.to_string()))]
```

Comment:
Just note that this is at INFO log level.

Our instruments are all like this, but I feel like we might want to downgrade to debug or trace. Not a blocker though.